modAionImpl/src/org/aion/zero/impl/sync/SyncStats.java

```diff
@@ -37,6 +37,10 @@
      */
     private final Map<String, Long> blocksByPeer = new HashMap<>();
 
+    private final Map<String, Long> importedByPeer = new HashMap<>();
```

Comment:
the value of the Map can be an Integer, cause Int max is a pretty big number for  blocks

Reply:
Don't need to change now. Please create a ticket for refactoring the variable use.

Reply:
created #825